### PR TITLE
DROOLS-2373 : BRL 'is contained in the (comma separated) list' does not produce multiselect dropdown

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2373

https://github.com/kiegroup/drools/pull/2189
https://github.com/kiegroup/kie-wb-common/pull/2359
https://github.com/kiegroup/drools-wb/pull/1025

### CODE REVIEW

**Comment by jomarko** [comment]
@Rikkola ping, any thoughts on my comments?

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/BRLConditionColumnPlugin.java [hunk 1]
```diff
@@ -193,6 +193,7 @@ boolean getDefinedVariables(RuleModel ruleModel) {
                                                                                  iv.getFactField());
             variable.setHeader(editingCol.getHeader());
             variable.setHideColumn(editingCol.isHideColumn());
+            variable.setOperator(iv.getOperator());
```
[the remark this answers is not in the document]
Comment (Rikkola) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/BRLConditionColumnPlugin.java]: Good idea. This reduces the amount of magic Strings in the constructor.

**Comment by Rikkola** [comment]
@jomarko Sorry, been busy with trying to get the other PRs I have to just compile ;) I think for this it is best I add the fix for templates in these commits. So I will take a look once I get my builds working again.

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/BRLConditionColumnPlugin.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rikkola) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/BRLConditionColumnPlugin.java]: My comment above means no sense when I read this again, but I guess what I means is it is better to use the bottom version.

**Comment by manstis** [comment]
@Rikkola Looks like `jacoco.exec` has crept in; can you delete [this](https://github.com/kiegroup/drools-wb/pull/1025/files#diff-5b0e518c603bc40c8c8b55cf6f84f7c4).

**Comment by Rikkola** [comment]
@manstis sure. I'll add a git ignore for it. For me it keeps getting picked up.

**Comment by jomarko** [comment]
@Rikkola @manstis hi, this seems to me as fine to be merged. Is there anything that I am missing and block us from merging? Asking because we have to create 7.14.x cherry pick until Friday.

**Comment by Rikkola** [comment]
@jomarko @manstis  nope, I was just busy with the other 7.2.1 bug and meetings yesterday. Let's merge this in and I'll prepare the cherry-pick for 7.14.x.

**Pull request merged**